doc/runtimes/runtimeDebianFlavor.md

```diff
@@ -0,0 +1,48 @@
+# Oryx runtime base images debian flavor
```

Comment:
Is this file autogenerated or one that we'll need to manually update?

Reply:
This file currently was manually updated, we'd like to have a doc keeping track of all debian flavors of base images.